js/ui/drawer/ui.drawer.rendering.strategy.overlap.js

```diff
@@ -44,17 +44,12 @@ class OverlapStrategy extends DrawerStrategy {
 
     _fixOverlayPosition($overlayContent) {
         // NOTE: overlay should be positioned in extended wrapper
-        const drawer = this.getDrawerInstance();
+        const position = ensureDefined(this._initialPosition, { left: 0, top: 0 });
+        translator.move($overlayContent, position);
 
-        if(typeUtils.isDefined(this._initialPosition)) {
-            translator.move($overlayContent, { left: this._initialPosition.left, top: this._initialPosition.top });
-        }
+        const drawer = this.getDrawerInstance();
         if(drawer.getDrawerPosition() === "right") {
             $overlayContent.css("left", "auto");
-
-            if(drawer.option("rtlEnabled")) {
-                translator.move($overlayContent, { left: 0 });
```

Comment:
Why we are going to remove this?

Reply:
Overlay-content should have left=0 if the panel is open, but left=content.width if it's closed. This special statement is not only unnecessary, but also breaks the behavior - it will always make right panel opened in RTL.

Reply:
ok